libs/domain/picking/src/services/adapter/picking-list-default.adapter.ts

```diff
@@ -154,6 +154,10 @@ export class PickingListDefaultAdapter implements PickingListAdapter {
     products: PickingProduct[]
   ): PickingList {
     const cardNote = data.pickingListItems[0].salesOrders[0].cartNote;
+    const orderReference =
+      data.pickingListItems[0].salesOrders[0].orderReference;
+    const requestedDeliveryDate: string =
+      data.pickingListItems[0].salesShipments[0].requestedDeliveryDate;
```

Comment:
Please remove this logic from offline adapter as it's now moved to here.

Reply:
there is no logic regarding `orderReference` and `requestedDeliveryDate` in `picking-list-offline.adapter.ts`. Or do you mean something different?

Reply:
Check offline adapter there is already a code that parses `requestedDeliveryDate` and entity also has it.
That should only be done in one place - either in offline or in default adapter but not in both.
I would keep it in offline feature because to support them in online mode we need BAPI to support filtering which it does not do yet so for now we can keep it as is in the offline adapter.